src/providers/bedrock.ts

```diff
@@ -765,6 +765,7 @@ export const AWS_BEDROCK_MODELS: Record<string, IBedrockModel> = {
   'meta.llama3-1-8b-instruct-v1:0': BEDROCK_MODEL.LLAMA3_1,
   'meta.llama3-70b-instruct-v1:0': BEDROCK_MODEL.LLAMA3,
   'meta.llama3-8b-instruct-v1:0': BEDROCK_MODEL.LLAMA3,
+  'meta.llama3-2-3b-instruct-v1:0': BEDROCK_MODEL.LLAMA3_2,
```

Comment:
Are you sure this model is available as is? If so we should probably add all of the other llama 3.2s and 3.3s as well. 

Related: https://docs.aws.amazon.com/bedrock/latest/userguide/inference-profiles-support.html